manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -92,34 +100,114 @@ public void init(Container container) throws Exception {
             }
         }
 
-        if(Files.isRegularFile(mapSettingsPath)){
-            mapConfig = loadMapSettingsJson();
-        }
+        // Will throw if failed, stopping startup fast and hard
+        loadMapSettingsJson();
+        loadManagerConfigJson();
     }
 
     @Override
     public void start(Container container) throws Exception {
-        /* code not overridden yet */
+
+        // Check if the configuration contains references to files that are located in the deployment, and if they are,
+        // move them to the storageDir.
+        ObjectNode changedConfig = this.checkAndFixImageReferences(this.managerConfig);
```

Comment:
This is mostly there as a "migration" script, as an automatic way to move the images when this code is first ran (e.g. when old projects get this feature). Because when running the manager with these changes for the first time, we are required to move the references so that next migrations don't delete the files.

Reply:
If the `changedConfig` variable is `null`, both the images and the manager_config are copied over to the storageDir right after that, so the migration of both the manager_config and its images occur at startup.

Reply:
But this would mean that currently-deployed projects will not be migrated to the `storageDir`. Would you like to add the feature as an env var or something? Because I thought that was part of the reason of this PR, so that users don't lose their configuration upon upgrading to a newer version.